shell.go

```diff
@@ -7,14 +7,17 @@ import (
 	"strings"
 )
 
-// CommandInShell sets up a new command with the environment set up by users environment.
+type staticString string
```

Comment:
I think the right term is "string literal" in this case. Maybe even a comment for the type would be useful if anyone is confused?

Reply:
Thanks